kite-data/kite-data-core/pom.xml

```diff
@@ -271,5 +271,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
```

Comment:
I'm confused (so I'm probably going to learn something new). This profile seems identical to the one above it except it has a different `id` and doesn't have the `install-hdp-classifier` execution. Do we need both?